tests/foreman/cli/test_contentview.py

```diff
@@ -647,6 +647,37 @@ def test_positive_create_composite(self):
 
     @tier2
     @run_only_on('sat')
+    def test_positive_remove_version_by_id_from_composite(self):
+        """Create a composite content view and remove its content version by id
+
+        @id: 0ff675d0-45d6-4f15-9e84-3b5ce98ce7de
+
+        @assert: Composite content view info output does not contain any values
+
+        @CaseLevel: Integration
+        """
+        # Create CV
+        comp_cv = make_content_view({
+            'composite': True,
+            'organization-id': self.org['id']
+        })
+        ContentView.publish({u'id': comp_cv['id']})
```

Comment:
Looks like we are publishing a blank( without any CV) composite content-view and removing that version ?
I think valid test would be like:
- create a composite cv
- add a CV( with some contents) to composite CV created in step1
- publish the composite cv
- remove published version

If there is already a test based on above steps. Then I'm fine with this test too but with updated docstring as it doesn't contain any contents or CV